src/master-playlist-controller.js

```diff
@@ -395,10 +378,6 @@ export class MasterPlaylistController extends videojs.EventTarget {
       this.tech_.trigger({type: 'usage', name: 'hls-aes'});
     }
 
-    if (Hls.Playlist.isFmp4(media)) {
-      this.tech_.trigger({type: 'usage', name: 'hls-fmp4'});
```

Comment:
This code was moved to `setupSegmentLoaderListeners_()` and we trigger the usage event when a `segmentLoader` triggers an `fmp4` event.